static/server/taskbar.css

```diff
@@ -37,14 +40,22 @@ li.ui-state-focus {
 	text-decoration: underline;
 }
 
-@media only screen and (min--moz-device-pixel-ratio: 1.5),
-only screen and (-o-min-device-pixel-ratio: 1.5/1),
-only screen and (-webkit-min-device-pixel-ratio: 1.5),
-only screen and (min-device-pixel-ratio: 1.5) {
-    #kiwixtoolbar button, #kiwixtoolbar input {
-    	font-size: 2em;
+/* the breakpoint "600px" is the width of the 3 btns "library", "home" and "randoml" */
+@media (max-width: 600px) {
+    .kiwixsearch {
+        float: none;
+        margin-top: 5px;
     }
+
     #kiwixsearchbox {
-		width: 7em;
-	}
```

Comment:
it does not handle pixel ratio, it only set a few css rules for devices with high pixels density screen. I dont think these rules are required, I will test a bit more